README.md

```diff
@@ -20,6 +20,7 @@ addon | version | summary
 --- | --- | ---
 [sale_automatic_workflow](sale_automatic_workflow/) | 9.0.1.0.0 | Sale Automatic Workflow
 [sale_exception](sale_exception/) | 9.0.1.0.0 | Custom exceptions on sale order
+[sale_product_set](sale_product_set/) | 9.0.1.0.0 | Sale product set
```

Comment:
Don't change this file. It's automatically changed. Please also squash the commits when doing to avoid conflicts.